.github/dependabot.yml

```
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  # maintain dependencies for GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly" # default = monday
+    open-pull-requests-limit: 5
+    labels:
+      - "dependencies"
+      - "github_actions"
+
+  # maintain dependencies for Gradle
+  - package-ecosystem: "gradle" # checks build.gradle(.kts) and settings.gradle(.kts)
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "weekly"
```
otherwise we'll get overwhelmed with dependabot stuff